### DISCUSSION TITLE
Import your own gguf model not possible anymore ?

### DISCUSSION THREAD

**Discussion by benjaminantunes** [open-webui/open-webui#1103] [General]
Hi,

It was possible to import our own gguf model, but it seems like since the update with LiteLLM, I cannot find this feature anymore. 
Was it removed ?

**Comment by justinh-rahb** [comment]
<img width="690" alt="Screenshot 2024-03-08 at 7 15 01 AM" src="https://github.com/open-webui/open-webui/assets/52832301/cf3970d3-0b2e-4ad9-9b20-d9d27d088b64">

**Reply by benjaminantunes** [reply]
Thank you for this, didn't notice that it was hidden !

**Comment by goodmancodes** [comment]
Doesn't seem to be working. You click the Show, it opens a part for you to click and select the model, but you click and nothing happens.

**Reply by justinh-rahb** [reply]
I get a file upload prompt. Please ensure you're running the latest version of WebUI, there were fixes to bugs related to file picker buttons very recently. Re-pull and re-run.

**Reply by shuasimodo** [reply]
Sorry to jump in here. For me the upload prompt sits at 0% for a while and does nothing. I just leave it for a few minutes and it will upload pretty fast. 

It's actually working, just progress status isn't in realtime it looks like. Depending on your cpu give it 5-10 min.
